### DISCUSSION TITLE
CSS - overflow hidden não está funcionando

### DISCUSSION THREAD

**Discussion by Al3xFlores** [frontendbr/forum#1722] [Perguntas e Respostas]
Olá devs, estou fazendo um projeto flappy bird porem estou enfrentando um problema de CSS.
O lnk a baixo mostra como está os canos verdes saindo para forá do "[wm-flappy]" e está centralizado sendo que éra para estarem na lateral esquerda: https://www.4shared.com/photo/GB4y9yMbiq/Capturar.html

resumindo os canos estão centralizados e saindo para fora,
segue o codigo CSS abaixo e link do repositorio : https://github.com/Al3xFlores/Aprendizado/tree/master/Flappy-Bird-Game

`* {
  box-sizing: border-box;
}

@font-face {
  font-family: 'Pixel';
  src: url('../fonts/Pixel.ttf');
}

html {
  height: 100%;
}

body {
  margin: 0;
  background-color: #0D262D;
  color: #fff;
  height: 100%;
}

.conteudo {
  display: flex;
  flex-direction: column;
  align-items: center;
  min-height: 100%;
  font-family: Pixel;
  font-size: 20px;
}

[wm-flappy] {
  position: relative;
  border: 5px solid dodgerblue;
  height: 500px;
  width: 1024px;
  margin: 15px;
  background-color: deepskyblue;
  overflow: hidden;
  
}

.passaro {
  position: absolute;
  width: 60px;
  left: calc(50% - 30px);
}

.par-de-barreiras {
  position: absolute;
  top: 0;
  height: 100%;
  display: flex;
  flex-direction: column;
  justify-content: space-between;
}

.barreira {
  display: flex;
  flex-direction: column;
  align-items: center;
}

.barreira .borda {
  height: 30px;
  width: 130px;
  background: linear-gradient(90deg, #639301, #A5E82E);
  border: 2px solid #000;
}

.barreira .corpo {
  height: 150px;
  width: 110px;
  background: linear-gradient(90deg, #639301, #A5E82E);
  border-left: 2px solid #000;
  border-right: 2px solid #000;
}`

**Comment by rafaelangical** [comment]
coloca o html + css em um repo ou gist que fica mais fácil pra ajudar.

**Comment by Al3xFlores** [comment]
> coloca o html + css em um repo ou gist que fica mais fácil pra ajudar.

Boa tarde @rafaelangical , vlw por me ajudar
segue o link do repositorio.

https://github.com/Al3xFlores/Aprendizado/tree/master/Flappy-Bird-Game

**Comment by glauberm** [comment]
O `.par-de-barreiras` precisar estar dentro do `[wm-flappy]` para o `overflow: hidden` funcionar como você quer

**Comment by glauberm** [comment]
Um codepen rápido pra te demonstrar: https://codepen.io/glauberm/pen/MWwVwQo

**Comment by Al3xFlores** [comment]
Nossa man que suceso você, funcionou  muito obrigado pela ajuda mesmo.
